src/Microsoft.TestPlatform.Common/Microsoft.TestPlatform.Common.csproj

```diff
@@ -6,11 +6,13 @@
   <Import Project="$(TestPlatformRoot)scripts/build/TestPlatform.Settings.targets" />
   <PropertyGroup>
     <AssemblyName>Microsoft.VisualStudio.TestPlatform.Common</AssemblyName>
-    <TargetFrameworks>netstandard2.0;netstandard1.3;net451</TargetFrameworks>
+    <TargetFrameworks>net6.0;netstandard2.0;netstandard1.3;net451</TargetFrameworks>
```

Comment:
We don't want to ship the net6.0, we simply add it to avoid duplicate definition in tests as they target net6.0 but Common didn't target netcoreapp3.0 so was always having the `NullableAttributes`.